common/app/views/support/AmpAd.scala

```diff
@@ -0,0 +1,43 @@
+package views.support
+
+import model.Tag
+import model.Article
+import play.api.libs.json.Json
+
+object AmpAd {
```

Comment:
Can we make this `AmpAd` classes instead of a singleton object?
ex: 
```
case class AmpAd(article: Article, uri: String, edition: String) {
    def toJson(): Json = ...
}
case class AmpAdDataSlot(article: Article) {
    def toString(): String = ...
```